post-office-stub/src/data/postOfficeResponse/postOfficeResponseNoName.ts

```diff
@@ -0,0 +1,13 @@
+export const POST_OFFICE_RESPONSE_NO_NAME =
+	{
+		"@timestamp": "2024-03-04T20:05:28.969Z",
+		"level": "FATAL",
+		"message": "Unhandled Exception: Cannot read properties of undefined (reading 'name')",
```

Comment:
So this is the error that comes from the frontend as a result of a response from PO that it just an empty array (meaning that when we are trying to access `postOfficeData[0].name` we get this error)